README.md

```diff
@@ -42,6 +42,22 @@ yarn add --dev react-native-svg-transformer
 
 ### Step 3: Configure the react native packager
 
+#### For Expo SDK v35.0.0 or newer
+If you are using [Expo](https://expo.io/) v35.0.0, you also need to add "ts" and "tsx" in sourceExts on `app.json`:
+
+```json
+{
+  "expo": {
+    "packagerOpts": {
+      "config": "metro.config.js",
+      "sourceExts": ["js", "jsx", "ts", "tsx", "svg"]
+    }
+  }
+}
+```
+
+---
+
 #### For React Native v0.57 or newer / Expo SDK v31.0.0 or newer
 
 Merge the contents from your project's `metro.config.js` file with this config (create the file if it does not exist already).
```

Comment:
I get an error saying that it can't find the metro.config.js.  I assume that I need to create this.  What does it need to contain?

Reply:
check here : https://github.com/kristerkari/react-native-svg-transformer#for-react-native-v057-or-newer--expo-sdk-v3100-or-newer ;)

Reply:
Unfortunately this just crashes Expo.

Reply:
> Unfortunately this just crashes Expo.

That's weird I'm sorry :s 
I made this pull request to work with SVG files imported on expo project. (The aim of this package)
For direct SVG drawing on component, I don't know if it's safe to use this package ?

Reply:
It's OK, I finally sorted out the problem, thanks.  Expo installs the wrong version of react-native-svg: it needs to be 9.9.2.